crossbar.json

```diff
@@ -162,19 +162,7 @@
                 "max_frame_size": {
                     "description": "Maximum size in bytes of incoming WebSocket frames accepted or 0 to allow any size. (default: 0)",
                     "type": "integer",
-                    "minimum": 0
-                },
-                "require_masked_client_frames": {
-                    "description": "Require all WebSocket frames received to be masked. (default: true)",
-                    "type": "boolean"
-                },
-                "apply_mask": {
-                    "description": "Actually apply WebSocket masking (both in- and outgoing). (default: true)",
-                    "type": "boolean"
-                },
-                "max_frame_size": {
-                    "description": "Maximum size in bytes of incoming WebSocket frames accepted or 0 to allow any size. (default: 0)",
-                    "type": "integer",
```

Comment:
why was this config removed?

Reply:
Because those 3 were duplicates. See commit message.